zcash_test_vectors/zip_0244_zsa.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
```

Comment:
file name is misleading. do we really define this is zip244?

Reply:
We could change the name. We define these changes in ZIP 226 and 227 both, maybe we call it "digests_zsa" or something?

Reply:
Moved and renamed.